libraries/ESP8266WebServer/examples/HttpAdvancedAuth/HttpAdvancedAuth.ino

```diff
@@ -0,0 +1,45 @@
+#include <ESP8266WiFi.h>
```

Comment:
Please add a public domain / CC0 license at the top of the sketch

Reply:
I copied this sketch from the HttpBasicAuth folder and it does not have a license ..
I am not sure of what should I put here ..
can you give an Example ??

Reply:
E.g. https://github.com/esp8266/Arduino/blob/master/libraries/esp8266/examples/Blink/Blink.ino#L1-L4

Reply:
Sorry, i mean public domain license, as in Blink example. It has been mentioned before that using LGPL license on examples is too restrictive. We already have a few such examples, and it would be better to migrate all the examples to CC0/public domain license.

Reply:
sorry I just copied the license from the next sketch in the folder .
fixed it now